Develop

MaxF1996 opened the pull request:
link to gulp-repo: https://github.com/MaxF1996/MyBike-Landing

volodymyr-soltys97 commented on README.md:
```diff
@@ -4,18 +4,21 @@
 - **DON'T** try to do it `Pixel Perfect` - implement it the most `simple` way so it looks similar;
 - When done check yourself using the [CHECKLIST](./checklist.md) when finished;
-- Deploy and create a Pull Request with a [DEMO LINK](https://<your_account>.github.io/layout_landing-page/)
+- Deploy and create a Pull Request with a [DEMO LINK](https://MaxF1996.github.io/layout_landing-page/)
```
Add the demo link to the PR description instead of repo link 
<img width="698" alt="image" src="https://github.com/user-attachments/assets/82375d6f-b31e-46ab-8b80-b1114e6d3a5d">

MaxF1996 replied:
There is this link - https://MaxF1996.github.io/layout_landing-page/ in readme right now

volodymyr-soltys97 reviewed (changes requested):
Good job 👍 
Let's improve your page
1. You need to show additional phone number when hovering this icon and make a call when user clicks here
<img width="115" alt="image" src="https://github.com/user-attachments/assets/35435f0d-e7cc-43c1-95ef-6c692f8c7d24">

2. Add hover effects for images 
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/cc38a796-967d-4965-a5d7-3ac8100cbc33">

3. Change the autofill styles for the form fields
https://css-tricks.com/snippets/css/change-autocomplete-styles-webkit-browsers/ 
<img width="577" alt="image" src="https://github.com/user-attachments/assets/353cbc59-b57f-472f-bff1-a63521f8f066">

4. Add hover effects here 
<img width="363" alt="image" src="https://github.com/user-attachments/assets/b7d13166-f0f3-4766-9fca-6917940a6dba">

5. Fix mail to address for this link 
<img width="200" alt="image" src="https://github.com/user-attachments/assets/9769d05f-682f-4197-a4bd-cb26f6ffc820">
<img width="160" alt="image" src="https://github.com/user-attachments/assets/52b9be29-5bec-4c2e-8e4e-c4336d82fb11">

MaxF1996 commented:
1. Added title attriburte
2. scale now
3. made looking like in figna (filled)
4. there was opacity hover effect - changed to increasing font-size
5. fixed
6. should I swap to DEMO LINK from gulp repo or leave fork DEMO ? Right now there is fork DEMO Link in readme

SanyaBratashchuk reviewed (changes requested):
1. make all fields required
